### PR TITLE
npm: make our React packages consistent

### PR DESCRIPTION
**Pull request by mimecuvalo**
something we noticed in https://github.com/tldraw/tldraw/pull/4475#discussion_r1756534369
that our packages weren't consistent for React. this unifies them.

### Change type

- [ ] `bugfix`
- [ ] `improvement`
- [ ] `feature`
- [ ] `api`
- [x] `other`

### CODE REVIEW

**Review by MitjaBezensek — APPROVED** [review]
💅

File: packages/tldraw/src/lib/ui/components/ZoomMenu/DefaultZoomMenu.tsx [hunk 1]
```diff
@@ -73,7 +73,6 @@ const ZoomTriggerButton = forwardRef<HTMLButtonElement, any>(
 						: 'tlui-zoom-menu__button__pct'
 				}
 				onDoubleClick={handleDoubleClick}
-				icon={breakpoint < PORTRAIT_BREAKPOINT.MOBILE ? 'zoom-in' : undefined}
```
Comment (MitjaBezensek) [inline packages/tldraw/src/lib/ui/components/ZoomMenu/DefaultZoomMenu.tsx:76]: Was this not needed?

**Review by MitjaBezensek — APPROVED** [review]

File: packages/tldraw/src/lib/ui/components/ZoomMenu/DefaultZoomMenu.tsx (hunk 1 above)
Comment (mimecuvalo) [reply packages/tldraw/src/lib/ui/components/ZoomMenu/DefaultZoomMenu.tsx:76]: yeah, this seems to be leftover from some previous refactor

**Comment by mimecuvalo** [comment]
@SomeHats i think the deploy error is maybe related to your fix here? https://github.com/tldraw/tldraw/pull/4555
i can't tell

**Pull request merged**